controllers/openstacknetconfig_controller.go

```diff
@@ -940,3 +992,44 @@ func (r *OpenStackNetConfigReconciler) ensureMACReservation(
 
 	return nodeMACReservation, nil
 }
+
+//
+// allMACReservations - get all reservations from static + dynamic created
+// also check if there are any dupe in OVNBridgeMacMappings.StaticReservations
```

Comment:
this doesn't look like checking for dups? also it always returns nil as error

Reply:
right, I moved the dupe check to the webhook. let me update this here